tshirts.js

```diff
@@ -9,8 +9,7 @@ function size(cms) {
         return 'L';
     }
 }
-
-expect(size(37)).equals('S');
+expect(size(38)).equals('S');
 expect(size(40)).equals('M');

```

Comment:
Adding a test case for size 42 will guarantee well defined behavior. Assume what your production code needs to do, and expect test cases on similar lines. That will make sure that the production code is intended to behave in a certain way.